packages/manager/modules/dbaas-logs/src/detail/inputs/index.js

```diff
@@ -0,0 +1,37 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ng-ovh-cloud-universe-components';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import 'ovh-api-services';
+import 'ovh-ui-angular';
+
+import 'ovh-ui-kit/dist/oui.css';
+
+import add from './add';
+import component from './logs-inputs.component';
+import console from './console';
```

Comment:
the console is a reserved word. can you rename?

Reply:
This is done :)